libs/persona2vec_gcn/tests/test.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import persona2vec_gcn as pvgcn
+from persona2vec_gcn.model import Persona2VecGCN
+from persona2vec_gcn.vgae import DeepVGAE
+import numpy as np
+import networkx as nx
+
+
+class TestPersona2VecGCN(unittest.TestCase):
+    def setUp(self):
+        self.G = nx.karate_club_graph()
+        self.A = nx.adjacency_matrix(self.G)
+
+    def test_embedding(self):
+        dimensions = 64
+        X = pvgcn.laplacian_eigenmap(self.G, dimensions=16)
+        model = Persona2VecGCN(self.G, X=X, lambd=0.1, dimensions=dimensions)
+        emb = np.vstack(model.embedding.values())
+        assert emb.shape[1] == dimensions
+        assert emb.shape[0] >= self.A.shape[0]
```

Comment:
1. Add an input argument for a node feature matrix for DeepVGAE and Persona2vecGCN

This is the proposed usage of the Persona2vecGCN, where `X` is the node feature matrix with shape (n_nodes, n_features). X is a numpy array. 

X can be the adjacency matrix, Laplacian eigenvectors, or node features from a dataset.